larpandoracontent/LArTwoDReco/TrackInEMShowerAlgorithm.h

```diff
@@ -121,17 +108,16 @@ class TrackInEMShowerAlgorithm : public pandora::Algorithm
          *  @param  startPoint the line start point
          *  @param  lineDirection the line direction unit vector
          */
-	    SortByDistanceAlongLine(const pandora::CartesianVector &startPoint, const pandora::CartesianVector &lineDirection) :
-            m_startPoint(startPoint), m_lineDirection(lineDirection.GetUnitVector()) {}
+	    SortByDistanceAlongLine(const pandora::CartesianVector &startPoint, const pandora::CartesianVector &lineDirection);
```

Comment:
Check indentation. Not a tab character? Can run the removeWhiteSpace.sh script too, to finish up.